flink-connectors/flink-sql-connector-hive-1.2.2/src/main/resources/META-INF/NOTICE

```diff
@@ -6,8 +6,8 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
+- io.airlift:aircompressor:0.8
 - org.apache.hive:hive-exec:1.2.2
 - org.apache.hive:hive-metastore:1.2.2
-- org.apache.thrift:libfb303:0.9.2
 - org.apache.orc:orc-core:1.4.3
-- io.airlift:aircompressor:0.8
+- org.apache.thrift:libfb303:0.9.2
```

Comment:
Note: There are more problems with this file: https://github.com/apache/flink/pull/12682